yogstation/code/modules/jobs/job_types/clerk.dm

```diff
@@ -21,6 +21,21 @@
 
 	changed_maps = list("EclipseStation", "OmegaStation")
 
+	mail_goodies = list(
+		/obj/effect/spawner/lootdrop/maintenance/three = 35, //bunch of stuff that could interest assistants
+		/obj/item/stack/sheet/plastic/five = 30,
+		/obj/effect/spawner/lootdrop/plushies = 20,
+		/obj/item/toy/minimeteor = 15,
+		/obj/item/circuitboard/computer/slot_machine = 15,
+		/obj/item/twohanded/dualsaber/toy = 10,
+		/obj/item/toy/windupToolbox = 10,
+		/obj/item/storage/box/fancy/cigarettes/cigpack_syndicate = 5,
+		/obj/item/storage/box/fancy/heart_box = 5,
+		/obj/item/lipstick/random = 5,
+		/obj/item/skub = 2, //pro skub have taken over the mail
+		/obj/item/stack/telecrystal = 1
```

Comment:
probably for the best if we don't give them TC, even if it's super rare and only 1
I'm willing to let it pass if another maint has a different opinion

Reply:
replaced with a WC, make a nice souvenir for any tourist buying it during their trip to NT state of the art spacestation